src/presentation/ui/page/marketplace.templ

```diff
@@ -0,0 +1,391 @@
+	@componentStructural.HorizontalTabHeader(
+		[]componentStructural.TabHeaderItem{
+			{Label: "Installed", Value: "installed"},
+			{Label: "Catalog", Value: "catalog"},
+		}, marketplaceOverview.ListType, "reloadMarketplacePageContent",
+	)
+	<div class="bg-os-800 -mt-4 rounded-b-xl rounded-r-xl p-4">
+		<div class="bg-os-500 rounded-b-lg rounded-r-lg p-6">
+			if marketplaceOverview.ListType == "installed" {
+				@MarketplaceInstalledItemsTable(marketplaceOverview.InstalledItemsList)
+			} else {
+				@MarketplaceCatalogItemsTab(marketplaceOverview.CatalogItemsList)
+			}
+		</div>
+	</div>
+}
+
+templ ScheduleMarketplaceItemInstallationForm(vhostsHostnames []string) {
+	<form
+		id="schedule-marketplace-item-installation-form"
+		hx-post="/api/v1/marketplace/catalog/"
+		hx-indicator="#loading-overlay"
+		hx-swap="none"
+		@submit="$store.main.refreshScheduledTasksPopover()"
+	>
+		<input type="hidden" name="id" :value="marketplaceItem.id"/>
+		<input type="hidden" name="dataFields" :value="dataFieldsAsString"/>
+		<div class="my-6 space-y-2">
+			<div class="w-100% inline-flex space-x-2">
+				<div class="w-60%">
+					@componentForm.SelectInput(
+						"hostname", "Hostname", "marketplaceItem.hostname",
+						"", vhostsHostnames, true,
+					)
+				</div>
+				<div class="w-40%">
+					@componentForm.InputFieldWithPrefix(componentForm.InputFieldWithPrefixDto{
+						Type:                "text",
+						Id:                  "urlPath",
+						Label:               "URL Path",
+						BindInputValuePath:  "marketplaceItem.urlPath",
+						BindPrefixValuePath: "hostnameWithTrailingSlash",
+						InfoDescription:     "Path where the respective installed item will be available for access.",
+					})
+				</div>
+			</div>
+			@componentStructural.CollapsibleSection("Install Parameters", "ph-brackets-curly") {
+				<template x-if="marketplaceItem.dataFields.length > 0">
+					<div class="grid grid-cols-2 gap-4">
+						<template x-for="dataField in marketplaceItem.dataFields">
+							<div>
+								<template x-if="dataField.type === 'text' || dataField.type === 'email'">
+									@componentForm.InputFieldClientSide(
+										"dataField.type", "dataField.label", "dataField.value",
+									)
+								</template>
+								<template x-if="dataField.type === 'password'">
+									@componentForm.PasswordInputClientSide(
+										"dataField.label", "dataField.value",
+									)
+								</template>
+								<template x-if="dataField.type === 'select'">
+									@componentForm.SelectInputClientSide(componentForm.SelectInputClientSideDto{
+										BindLabel:     "dataField.label",
+										BindModelPath: "dataField.value",
+										BindOptions:   "dataField.options",
+									})
+								</template>
+							</div>
+						</template>
+					</div>
+				</template>
+				<template x-if="marketplaceItem.dataFields.length === 0">
+					<label class="italic text-slate-500">This marketplace item has no install parameters.</label>
+				</template>
```

Comment:
I would just not show anything. This may spark doubts on users on what this message means. If you want to keep it, add a phrase saying "you just need to click install" or something else.

Reply:
I decided to simply hide the section when there are no installation parameters, so I don’t even need to worry about displaying any div or message!